src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/TestData.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes a test data object")
+class TestData(
+  @Schema(description = "The key", example = "A")
+  val key: String = "",
+
+  @Schema(description = "The value", example = "AAAAA")
+  val value: String = "",
```

Comment:
Though there is hardly a difference between the Entity and the Model for this test object, I've kept them separate so that where difference do exist between how an object is stored and how it is used in services, we have the means of doing this, and for translating between them. This separation has come in extremely handy in other services.